### PR TITLE
Name some sub_hex functions

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

File: src/openrct2/interface/window.c [hunk 1]
```diff
@@ -541,7 +541,7 @@ static bool sub_6EA934(sint32 x, sint32 y, sint32 width, sint32 height)
  * @param width (bx)
  * @param height (cx)
  */
-static bool sub_6EA95D(sint32 x, sint32 y, sint32 width, sint32 height)
+static bool windows_fits_between_others(sint32 x, sint32 y, sint32 width, sint32 height)
```
Comment (rwjuk) [inline src/openrct2/interface/window.c]: Should this be plural? Does it handle multiple windows?

Comment (Gymnasiast) [inline src/openrct2/interface/window.c]: It should indeed be singular. It's a typo.

Comment (IntelOrca) [inline src/openrct2/interface/window.c]: Why not, you would say *this window fits*, wouldn't you?

File: src/openrct2/interface/window.c [hunk 2]
```diff
@@ -523,13 +523,13 @@ static bool sub_6EA8EC(sint32 x, sint32 y, sint32 width, sint32 height)
  * @param width (bx)
  * @param height (cx)
  */
-static bool sub_6EA934(sint32 x, sint32 y, sint32 width, sint32 height)
+static bool window_fits_within_space(sint32 x, sint32 y, sint32 width, sint32 height)
 {
 	if (x < 0) return false;
 	if (y < 28) return false;
```
Comment (marijnvdwerf) [inline src/openrct2/interface/window.c]: What's this 28? Top toolbar height?

Comment (IntelOrca) [inline src/openrct2/interface/window.c]: Yes

Comment (Gymnasiast) [inline src/openrct2/interface/window.c]: That could very well be the case. I'll put it the magic numbers issue.

**Pull request merged**